components/Common/BlogPostCard/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { useMemo } from 'react';
+import type { ComponentProps, FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import AvatarGroup from '@/components/Common/AvatarGroup';
+import Preview from '@/components/Common/Preview';
+import { shortHumanReadableDate } from '@/util/shortHumanReadableDate';
+
+import styles from './index.module.css';
+
+type Author = {
+  fullName: string;
+  src: ComponentProps<typeof AvatarGroup>['avatars'][number]['src'];
+};
+
+export type BlogPostCardProps = {
+  title: ComponentProps<typeof Preview>['title'];
+  type: Required<ComponentProps<typeof Preview>>['type'];
+  description: string;
+  authors: Author[];
+  date: Date;
+};
+
+const BlogPostCard: FC<BlogPostCardProps> = ({
+  title,
+  type,
+  description,
+  authors,
+  date,
+}) => {
+  const avatars = useMemo(() => {
+    const parsedAvatars = authors.map(({ fullName, src }) => ({
+      alt: fullName,
+      src,
+      toString() {
+        return fullName;
+      },
+    }));
+
+    return parsedAvatars;
+  }, [authors]);
+
+  const authorNames = avatars.join(', ');
```

Comment:
Does this need to be a variable on its own?

Reply:
I created it so that its intended usage was more clear, but if you think it's unnecessary, I can remove it.